csrc/scheduler/resize.cpp

```diff
@@ -558,7 +558,7 @@ void ResizeScheduler::schedule(Fusion* fusion, const HeuristicParams* params) {
   // only the remaining loop domain is propagated.
   if (repeat_id_moved_to_outermost) {
     const auto& [tvs_with_repeat_id, tvs_without_repeat_id] = partitionTvsById(
-        fusion->allTvs(),
+        fusion->allTvs();
```

Comment:
```suggestion
        fusion->allTvs(),
```
[Suggested by AI] Verified that the build will pass with this change.